ldapauthenticator/ldapauthenticator.py

```diff
@@ -277,14 +304,18 @@ def getConnection(userdn, username, password):
                     username=username,
                     userdn=userdn
             ))
+
+            #if login is an email and alternative_username is required, use the username to login instead of the DN
+            eff_username = userdn if not self.alternative_username else username
```

Comment:
It seems this would mean that the user is authenticated as `self.alternative_username` no matter who actually logs in? In general the password the user supplies wouldn't correspond to the correct passworkd for `self.alternative_username` so I'm not sure how this would work?